leptonic/src/chip.rs

```diff
@@ -1,7 +1,7 @@
 use std::fmt::{Display, Formatter};
 
 use leptos::{
-    leptos_dom::{Callable, Callback},
+    Callable, Callback,
```

Comment:
`Callable` and `Callback` are included in `leptos::*` by default now, so these can be removed (in this file and all the others)!